sap/note/db.go

```diff
@@ -26,31 +26,20 @@ func (paging LinuxPagingImprovements) Initialise() (Note, error) {
 		SysconfigPrefix:             paging.SysconfigPrefix,
 		VMPagecacheLimitMB:          vmPagecach,
 		VMPagecacheLimitIgnoreDirty: vmIgnoreDirty,
+		UseAlgorithmForHANA:         true,
 	}, nil
 }
 func (paging LinuxPagingImprovements) Optimise() (Note, error) {
 	newPaging := paging
-	//conf, err := txtparser.ParseSysconfigFile(path.Join(newPaging.SysconfigPrefix, "/etc/sysconfig/saptune-note-1557506"), false)
 	conf, err := txtparser.ParseSysconfigFile(path.Join(newPaging.SysconfigPrefix, "/usr/share/saptune/notes/1557506"), false)
 	if err != nil {
 		return nil, err
 	}
 	inputEnable := conf.GetBool("ENABLE_PAGECACHE_LIMIT", false)
 	inputOverride := conf.GetInt("OVERRIDE_PAGECACHE_LIMIT_MB", 0)
-	inputIsHANA := conf.GetBool("TUNE_FOR_HANA", false)
 
-	if inputIsHANA {
-		// For HANA: new limit is 2% system memory
-		newPaging.VMPagecacheLimitMB = system.GetMainMemSizeMB() * 2 / 100
-	} else {
-		// For NW: new limit is 1/16 of system memory, within range 512 to 4096
-		newPaging.VMPagecacheLimitMB = system.GetMainMemSizeMB() / 16
-		if newPaging.VMPagecacheLimitMB < 512 {
-			newPaging.VMPagecacheLimitMB = 512
-		} else if newPaging.VMPagecacheLimitMB > 4096 {
-			newPaging.VMPagecacheLimitMB = 4096
-		}
-	}
+	// For HANA: new limit is 2% system memory
+	newPaging.VMPagecacheLimitMB = system.GetMainMemSizeMB() * 2 / 100
```

Comment:
This applies only for HANA, or for all? As the `if` was removed, I am assuming that it applies generally for HANA and NW and the comment needs adjustment...

Reply:
Yes, you are right. I will change the comment to make it clear. I had written the information to the commit comment, but totally forgot the comment in the source

Reply:
see new commit https://github.com/SUSE/saptune/pull/7/commits/efe1570014676a635fad9d4ae3ea2646dfaea3f1